app/views/layouts/_menu.html.erb

```diff
@@ -13,7 +13,10 @@
           <%= link_to "Employees", employees_path, class: "nav-link #{controller_name == "employees" ? 'active' : '' }" %>
         </li>
         <li class="nav-item">
-          <%= link_to "Company_branches", company_branches_path, class: "nav-link #{controller_name == "company_branches" ? 'active' : '' }" %>
+          <%= link_to "Company branches", company_branches_path, class: "nav-link #{controller_name == "company_branches" ? 'active' : '' }" %>
```

Comment:
I recommend you use single quotes when you use strings on ruby.

Reply:
changed